src/components/Advanced/TextBox/index.js

```diff
@@ -39,7 +39,7 @@ class TextBox extends Component {
 
   componentDidUpdate(prevProps, prevState) {
     // Changing dependencies props
-    if (this.props.dependencies) {
+    if (this.props.dependencies && prevProps.dependencies != undefined) {
```

Comment:
check for all falsy values, not just undefined, same as before &&

Reply:
Can just do `if (this.props.dependencies && prevProps.dependencies)` due to the wonders of javascript